met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -21,12 +21,12 @@ const AuthenticatedRoutes = () => {
             <Route path="/" element={<LandingPage />} />
             <Route path="/survey/listing" element={<SurveyListing />} />
```

Comment:
and let's remove 'listing' and go for /surveys instead of /survey/listing

Reply:
ok perfect will do